### DISCUSSION TITLE
Faikin Energy Consumption

### DISCUSSION THREAD

**Discussion by Rev-777** [revk/xESP32-Faikout#182] [Q&A]
Hi all,

I've just installed 5 Faikin boards into my Daikin split system. It's working great and providing tremendous control and (sometimes too much) data. For reference, FTXR18WVJUW9 & FTXR09WVJUW9 Emura series, and 3x FTXS09WVJU serviced by a 5MXS48WVJU9 outdoor unit.

Quick question, is it possible for individual energy consumption of the outdoor and interior units to be exposed to MQTT? It would be great to be able to see what sort of usage they're using and add individual entities to Home Assistant.

Thanks again for a great project. It added in seamlessly, was a fraction of the price Daikin wanted, and mostly importantly completely local.

![Image 2024-01-13 at 11 47](https://github.com/revk/ESP32-Faikin/assets/49879127/74bb823f-8a59-4567-bea4-a90f92800d84)

**Comment by revk** [comment]
Someone else's asked about this and we got a tad stuck on what the units were on the data is some S21 messages. Suggestions welcome.

**Reply by Rev-777** [reply]
Thanks, hopefully there is a solution to this someday.

**Reply by ando2040** [reply]
Can you post the HA YAML for your Fan and Liquid mini-graph cards please? They look neat but I can't make mine look like them.

**Reply by Rev-777** [reply]
Will send.

**Reply by revk** [reply]
That is not S21 messages!

**Reply by Rev-777** [reply]
Fair enough, I’ll remove and DM.

Still hoping for S21 power consumption messages if somebody were able to figure out. Much appreciated!

**Reply by revk** [reply]
Someone did the message but not the units/values. I can add something but ideally need more details.

**Reply by 44616E** [reply]
Check the issue, its been worked out and is now in beta!

**Reply by Rev-777** [reply]
Thanks, updated one of my splits. What else is needed to have these show as entities in HA?

edit: I noticed the release notes for 2024-03-30 "Added a Wh meter report (S21), no HA config, etc, yet.". Looking forward to having it exposed to HA.
